10LessonTen/Checkpoint2/index.html

```diff
@@ -69,6 +69,44 @@
       <div class="box4">
 
       </div>
+      <footer>
+        <div class="left">
+         <h2>Products</h2>
+         <ul>
+           <li>Burritos</li>
+           <li>Bowls</li>
+           <li>Quesadillas</li>
+           <li>Nachos</li>
+         </ul>
+        </div>
+        <div class="center">
+          <h2>Locations</h2>
+          <ul>
+            <li>San Antonio</li>
+            <li>Dallas</li>
+            <li>Mexico City</li>
+            <li>Houston</li>
+          </ul>
+        </div>
+        <div class="right">
+          <h2>Careers</h2>
+          <ul>
+            <li>Burrito land</li>
+            <li>Burrito Corp</li>
+            <li>Burritio Finance</li>
+            <li>Burrito XL</li>
+          </ul>
+        </div>
+        <div class="right1">
+          <h2>social media</h2>
+          <ul>
+            <li>Burrito land</li>
+            <li>Burrito Corp</li>
+            <li>Burritio Finance</li>
+            <li>Burrito XL</li>
+          </ul>
+        </div>
+      </footer>
```

Comment:
This website looks great. I like the overall feel and look of it. Your HTML markup and CSS are easy to read and understand. I did notice that you have a header tag with no content in it. If this is the case, that header tag (and the closing header tag) can be removed. Keep up the good work!